src/main/java/codechicken/nei/api/ItemInfo.java

```diff
@@ -458,8 +460,10 @@ else if (item == Items.fishing_rod || item == Items.flint_and_steel || item == I
     }
 
     private static void addSpawnEggs() {
-        addEntityEgg(EntitySnowman.class, 0xEEFFFF, 0xffa221);
-        addEntityEgg(EntityIronGolem.class, 0xC5C2C1, 0xffe1cc);
+        if (!Loader.isModLoaded("hodgepodge") || !TweaksConfig.addModEntityStats) {
```

Comment:
This isn't compatible with older versions of hodgepodge -- please add version identification or reflection to get this field

Reply:
True, added. I set 2.5.44 as min version for now but I don't know in which HP version will include the PR. I need to change the version in `dependencies.gradle` anyway (currently uses -pre version), at that time I would change the version here too.

Reply:
Can you just only run this if hodgepodge of a specific version is present?  I don't want to add a requirement for a specific version of hodgepodge in NEI; past what lwjgl3ify/etc would force.  
`if(hodgepodge && hodgehodge.version > xxxx) {registerStuff)`

Mainly if you made the check for `addModEntityStats` use reflection, and default to false, this check here would work perfectly